vizro-core/tests/unit/vizro/tables/test_dash_ag_grid.py

```diff
@@ -50,76 +48,20 @@ def test_dash_ag_grid(self):
         )
         # skipping only dashGridOptions as this is mostly our defaults for data formats, and would crowd the tests
 
-
-class TestCustomDashAgGrid:
```

Comment:
`class TestCustomDashAgGrid` tests are restructured into:
 - the `test_custom_dash_ag_grid` test in this file and
 - two new tests under the `class TestDunderMethodsAgGrid` inside the `test_ag_grid.py`.
 
Same for `class TestCustomDashDataTable` tests.